sam2/utils/misc.py

```diff
@@ -60,7 +60,7 @@ def get_connected_components(mask):
     """
     from sam2 import _C
```

Comment:
There is no _C directory in the sam2 folder in the first place right? Will this be generated on running. Is there support for video segmentation with cpu

Reply:
The `_C` module is compiled and generated with the [setup.py](https://github.com/facebookresearch/segment-anything-2/blob/main/setup.py#L40-L52) script

Reply:
Thank you